ledger/participant-state/src/main/scala/com/daml/ledger/participant/state/v1/ReadService.scala

```diff
@@ -79,17 +89,9 @@ trait ReadService extends ReportsHealth {
     * - *rejection finality*: if there is a [[Update.CommandRejected]] update
     *   with [[SubmitterInfo]] `info`, then there is no later
     *   [[Update.TransactionAccepted]] with the same associated [[SubmitterInfo]]
-    *   `info`. Note that in contrast to *no duplicate transaction acceptance*
+    *   `info`. Note that in contrast to *command deduplication*
     *   this only holds wrt the full [[SubmitterInfo]], as a resubmission of a
```

Comment:
This section is not very precise. Neither `deduplicateUntil` nor `maximumRecordTime` can be used to uniquely identify submissions.